Added es language support.

sergiodxa opened the pull request.

philbooth commented on package.json:
```diff
@@ -42,6 +43,7 @@
         "build-enbr": "./build.js -l en,br",
         "build-enda": "./build.js -l en,da",
         "build-ende": "./build.js -l en,de",
+	"build-enes": "./build.js -l es,es",
```
Typo here, you've got `es,es` instead of `en,es`.

And while you're doing that, could you also change the indentation to match the other lines (and on line 36 too)?

philbooth commented on src/languages/es.js:
```diff
@@ -0,0 +1,21 @@
+es: {
```
Can you change the indentation in this file so that it matches the other language modules? That way, the built files in `lib` will get sane indentation too, thanks.

philbooth commented on src/languages/es.js:
```diff
@@ -0,0 +1,21 @@
+    minute: [ 'minuto', 'minutos' ],
+
+    past: function (vagueTime, unit) {
+        return 'hace ' vagueTime + ' ' + unit;
```
You're missing a `+` here, causing the linter and the tests to fail.

philbooth commented:
Thanks heaps for this @sergiodxa!

If you can fix the minor issues that I raised inline, I'll be happy to get this merged and released. :smile:

philbooth commented:
Oh, we're also missing some new unit tests. One for the `just now` case, one for a case in the past and one for a case in the future. See the other languages for examples, e.g.:

* https://github.com/philbooth/vagueTime.js/blob/master/test/vagueTime.js#L414-L431
* https://github.com/philbooth/vagueTime.js/blob/master/test/vagueTime.js#L556-L564

sergiodxa commented:
I fixed the indentation, the typos, and added some test for recién (just now), dentro de poco (soon), hace 1 minuto (1 minute ago) and en 1 año (in 1 year).

Also, I added a pretest script to run ```npm run build```, only because I'm lazy to run ```npm run build``` and ```npm test``` every time.

philbooth commented:
Awesome stuff, published as version 1.3.0, thanks! :grin: :shipit:
